maze_project.iml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All the binary files (*.class) are garbage, remove

Reply:
Done

Reply:
V